hedera-node/configuration/dev/application.properties

```diff
@@ -14,3 +14,5 @@ bootstrap.throttleDefsJson.resource=genesis/throttles-dev.json
 ledger.id=0x03
 blockStream.streamMode=BOTH
 nodes.enableDAB=true
+hedera.recordStream.logDir=data/recordStreams
```

Comment:
`hedera.recordStream.logDir=data/recordStreams` Is this needed for any tests ?

Reply:
This is where the override in `ProcessUtils` should be. I put it in for parity, but it's not required